src/website/tracker/settings.py

```diff
@@ -336,6 +336,9 @@ def get_secret(name: str, encoding: str = "utf-8") -> str:
 GH_SECURITY_TEAM = "security"
 # Repository to post issues to
 GH_ISSUES_REPO = "nixpkgs"
+# Labels to set automatically on created Github issues, allowing users to filter
+# them.
+GH_ISSUES_LABELS = []
```

Comment:
Please add to the DjangoSettings instead so we can configure it uniformly. Everything beneath the TODO in that file is legacy and I'll clean it up eventually.